test/parallel/test-fs-read-parameters-errors.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const common = require('../common');
+const fixtures = require('../common/fixtures');
+const fs = require('fs');
+const filepath = fixtures.path('x.txt');
+const fd = fs.openSync(filepath, 'r');
+const length = 4;
+
+[true, null, undefined, () => {}, {}].forEach((value) => {
+  common.expectsError(() => {
+    fs.read(value,
+            Buffer.allocUnsafe(length),
+            0,
+            length,
+            0,
+            common.mustNotCall());
+  }, { code: 'ERR_INVALID_ARG_TYPE', type: TypeError });
```

Comment:
It would be nice to test for the error message as well to make sure the right error is triggered.